src/features/apiDelegation/offered/ChooseOrgPage/ChooseOrgPage.tsx

```diff
@@ -220,6 +222,16 @@ export const ChooseOrgPage = () => {
       >
         <PageHeader icon={<ApiIcon />}>{t('api_delegation.give_access_to_new_api')}</PageHeader>
         <PageContent>
+          {chosenApis.length < 1 && (
+            <RestartPrompter
+              spacingBottom
+              restartPath={
+                '/' + ApiDelegationPath.OfferedApiDelegations + '/' + ApiDelegationPath.ChooseApi
+              }
+              title={t('common.an_error_has_occured')}
+              ingress={t('api_delegation.delegations_not_registered')}
+            />
+          )}
```

Comment:
Yeah thought about having it but it was a bit "smør på flesk" since the users are already able to click on previous button and remove them or go to confirmation page and remove them. It's also like you say that it will be a problem when user has chosen many APIs.

I'll add it if UX says so after design review.

Reply:
Yeah, I see your point.
But I think this is such a significant change that you should probably ask UX beforehand.

Reply:
That's not the work process we use right now. And regardless it's not really a change to that page. I didn't delete anything or change any code related to that. But if you want to ask them, I think it's better that you do it because I'll be gone for 2 weeks from saturday and I think you will need to have design meeting next week in order to agree on something.

Reply:
Arbeidsprosessen vi ble enige om med UX i møte med dem tidligere er at: Vi skriver endringene i design review i issuet, og deretter kan UX kommentere hvis de er uenige i hvordan noe er løst, så fikser vi det. Vi er seniorutviklere og når vi jobber som det bør vi ikke trenge å spørre dem på forhånd for hver eneste avgjørelse vi tar. I tillegg sier Altinn at vi skal ta beslutninger på så lavt nivå som mulig som er det vi vil gjøre i dette tilfellet. Jeg tror heller ikke de vil at vi skal mase de opp og ned hele tiden siden de allerede er fylt til randen med arbeid å gjøre (Hvis de hadde villet at vi skulle spørre mer så hadde de sagt det). 

Jeg vet at jeg vil at det skal funke som jeg har gjort det i PR-en her og da er det ikke nødvendig å starte en lang prosess med dem når de kanskje er enige i det jeg allerede har gjort uansett (Det vil design review vise). Ja greit nok at vi hadde den lista du nevner når flowen var ChooseOrg => ChooseApi. Hvis jeg ikke visste hvordan jeg ville ha det, ville det være fornuftig å spørre dem.

Det er heller ikke noe problematisk å legge til denne oppgaven senere dersom de sier at de vil ha den featuren tilbake igjen da det er en standalone feature. Jeg vil da også få leverage til å foreslå at vi bytter ut lista der med CollectionBar siden det ikke eksisterer noen liste der i det hele tatt.

Du kan for all del jobbe som du vil med dine issues og spørre om ting som det her hele tiden men det er ikke noe jeg kommer til å blande meg opp i. Når du har tatt en avgjørelse og du har argumenter for å gjøre det du har lyst til så kommer jeg til å la det passere og sende det videre til systemet og design review.

Reply:
Det har ikke noe med at dette ikke vil fungere, det handler om samarbeid internt i teamet.

Avklaringer som dette skal taes på lavest mulig nivå, ja. Men det betyr ikke at vi ikke skal snakke sammen i teamet og bruke hverandres kompetase. Vi er et team av en grunn, ikke bare enkeltsående individer.

Det trenger ikke å gjøres til en stor greie, men når vi tenker vi bør endre på en slik ting så bør vi ihvertfall si ifra at vi tenker å gjøre det. Alt trenger selvfølgelig ikke å taes opp i plenum (ikke superenkle ting som ending i padding f.eks.), men å fjerne en feature fra en side i flyten synes jeg at de fortjener å få en headsup på før PRen går igjennom. Det er en kjapp ting å sende dem en kort melding.
Det er er ikke at akkurat denne endringen her er en stor ting, men siden vi har ukentlige releaser og APIdelegering er i prod så bør vi jo ikke gjøre det til en vane å dytte ut endringer i prod som så rulles tilbake uken etter.

Det skaper unødvendig støy, tenker jeg. Unødvendig nettop fordi det kan forhindres ved å sende en liten melding.

Jeg synes det er viktig at vi holder kommunisjonskanalene åpne mellom utviklere og UX. Det vil gagne oss, dem, og produktet som helhet. Og slike ting som dette kan avklares på 3 minutter hvis vi bare spør dem. Ikke minst det vil også skape større tillit og godvilje mellom oss. Noe som vi har sett at vi trenger mer av.

Jeg spurte dem selv, og de var enig i at det ikke var nødvendig, så da har vi fått den avklaringen. Det tok ikke lang tid og da slipper vi å bruke krefter på å lure på om de kommer til å mislike det når de ser det. Vi har da minimert risikoen for merarbeid senere.

Reply:
Der er vi uenige. Jeg syns det skaper unødvendig støy å spørre om den minste lille ting hele tiden mens du syns det skaper støy å ikke gjøre det. På dine PR-er så står du fritt til å spørre så mye du vil Alexandra men når jeg har gjort PR-en så kan ikke du bestemme over meg at jeg skal spørre dem, det er kontrollerende. Da får du heller spørre selv. 

Jeg tenkte at de kom til å synes det var smør på flesk jeg også og det viste seg jo at de gjorde (Ergo, det var ikke nødvendig å spørre. Akkurat som jeg trodde). Det blir ikke flyt i teamet av å spørre om den minste lille ting man lurer på (og hvertfall ikke flyt av at du skal bestemme hva andre skal spørre om), det blir flyt av at man gjør en ting også får man evt. Tilbakemelding om man skal gjøre det annerledes.

Når det kommer til hvordan de får beskjed om endringen så vil jo dette stå som endringer på issuen så jeg skjønner ikke hva du prøver å fremtvinge her.